src/deepsparse/benchmark_model/README.md

```diff
@@ -0,0 +1,165 @@
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+# Benchmarking
+
+The command-line (CLI) tool `deepsparse.benchmark` is meant to simplify benchmarking and be the central codepath for benchmarking the engine on ONNX or SparseZoo models.
+
+## Quickstart
+
+After you have installed `deepsparse` to your Python environment, the benchmark tool is available on your CLI. It is simple to run expressive model benchmarks on the DeepSparse Engine with minimal parameters.
+
+To run an sparse FP32 MobileNetV1 at batch size 16 for 10 seconds for throughput using 8 streams of requests:
+
+```
+deepsparse.benchmark zoo:cv/classification/mobilenet_v1-1.0/pytorch/sparseml/imagenet/pruned-moderate --batch_size 8 --time 10 --scenario async --num_streams 8
+```
+
+To run a sparse quantized INT8 6-layer BERT at batch size 1 for latency:
+
+```
+deepsparse.benchmark zoo:nlp/question_answering/bert-base/pytorch/huggingface/squad/pruned_quant_6layers-aggressive_96 --batch_size 1 --scenario sync
+```
+
+## What does it do?
+
+The tool will parse the arguments, download/compile the network into the DeepSparse Engine, generate input tensors, and execute the model depending on the chosen scenario. By default, it will choose a multi-stream or asynchronous mode to optimize for throughput.
+
+The number of executions is a result of setting the time duration to run executions for, set by the `-t` argument in seconds.
+
+### Synchronous (Single-stream) Scenario
+
+Set by the `--scenario sync` argument, the goal metric is latency per batch (ms/batch). This scenario submits a single inference request at a time to the engine, recording the time taken for a request to return an output. This mimics an edge deployment scenario.
+
+The latency value reported is the mean of all latencies recorded during the execution period for the given batch size.
+
+### Asynchronous (Multi-stream) Scenario
+
+Set by the `--scenario async` argument, the goal metric is throughput in items per second (i/s). This scenario submits `--num_streams` concurrent inference requests to the engine, recording the time taken for each request to return an output. This mimics a model server or bulk batch deployment scenario.
+
+The throughput value reported comes from measuring the number of finished inferences within the execution time and the batch size.
+
+
+## Usage
+
+In most cases good performance will be found in the default options so it can be as simple as running the command with a SparseZoo model stub or your local ONNX model.
```

Comment:
```suggestion
In most cases, good performance will be found in the default options so it can be as simple as running the command with a SparseZoo model stub or your local ONNX model.
```